.github/workflows/e2e_creator_learner_dashboard_and_editor_tabs.yml

```diff
@@ -60,9 +60,9 @@ jobs:
           VIDEO_RECORDING_IS_ENABLED: 0
       - name: Run Learner Dashboard E2E Test
         if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
-        run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-build --skip-install --suite="learnerDashboard" --prod_env
+        run: for run in {1..12}; do xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_e2e_tests --skip-build --skip-install --suite="learnerDashboard" --prod_env; done
```

Comment:
### PLEASE NOTE REVIEWERS: I will be removing this after the whole PR gets approved. Because in this way everytime I will push the changes, it will run the tests 12 times for testing.